src/clims/services/workflow/camunda/client.py

```diff
@@ -159,22 +159,35 @@ def unsafe_delete_all_deployments(self):
 
     def _refine_xml(self, tree, namespace):
         """
-        Adds a fully qualifed name to relative ones
+        Adds a fully qualifed name to relative ones. This ensures that all IDs that require
+        it will always have a fully qualified name so they will not clash with other plugins.
+
+        Example: User creates a diagram with the ID "SequenceSimple" and then registers it
+        through the plugin clims.plugins.demo.dnaseq, the process will actually be registered
+        as clims.plugins.demo.dnaseq.SequenceSimple. Furthermore, all calls to to subprocesses
+        will be made to fully qualified names using the same module.
+
+        Form keys are also changed to the fully qualified version.
+
+        If the user does fully qualify names (they contain a dot), no change takes place.
```

Comment:
"ID" and "formKey" are here used interchangeable, although it's only "formKey" that are handled in this method. I'm not certain of what the purpose of formKey is. It seems to be an identifier of the workflow. But what about the "form" in the formKey?

Reply:
A formkey is a Camunda feature that has the id of the form (view) that should be used in the UserTask (https://docs.camunda.org/manual/7.8/user-guide/task-forms/). We need some way to connect the UserTask and the view we're showing. We could also use the UserTask ID, but that would mean that we couldn't reuse views for different user tasks, because those IDs need to be unique.

Reply:
:+1: